### PR TITLE
Scrollable item info text in crafting menu

### PR DESCRIPTION
**Pull request by OzoneH3**

### CODE REVIEW

**Comment by Zireael07** [comment]
Great!

**Comment by mugling** [comment]
nice

**Comment by Night-Pryanik** [comment]
At last!

File: src/crafting_gui.cpp [hunk 1]
```diff
@@ -139,15 +140,15 @@ const recipe *select_crafting_recipe( int &batch_size )
...
     const int dataHalfLines = dataLines / 2;
     const int dataHeight = TERMY - ( headHeight + subHeadHeight );
+    const int infoWidth = width - FULL_SCREEN_WIDTH - 1;
 
     int lastid = -1;
 
     WINDOW *w_head = newwin( headHeight, width, 0, wStart );
     WINDOW *w_subhead = newwin( subHeadHeight, width, 3, wStart );
     WINDOW *w_data = newwin( dataHeight, width, headHeight + subHeadHeight, wStart );
+    WINDOW *w_iteminfo = newwin( dataHeight - 3, infoWidth, headHeight + subHeadHeight, wStart + width - infoWidth );
```
Comment (BevapDin) [inline src/crafting_gui.cpp]: This new window is never deleted.

Comment (kevingranade) [inline src/crafting_gui.cpp]: Whoops, missed that, PR shortly.

**Pull request merged**